src/melee/ft/chara/ftmario.c

```diff
@@ -160,3 +181,134 @@ s32 func_800E0CE0(s32 arg0) {
 
     return lbl_803C5A20[offset - 0xe];
 }
+
+//https://decomp.me/scratch/cdQ5o
+void func_800E0D1C(HSD_GObj* gobj) {
```

Comment:
This needs to return something, as used by drmario:

https://github.com/doldecomp/melee/blob/ae0c84368812db6f4f8dc403e0881ce3ab1fdf47/src/melee/ft/chara/ftdrmario_2.c#L13-L14

I think if you just `return r3` it will still match

Reply:
Got it, thanks for the catch :)